### DISCUSSION TITLE
[Script request] Mylar3

### DISCUSSION THREAD

**Discussion by spocoeric** [community-scripts/ProxmoxVE#312] [Request script]
### Application Name

Mylar3

### Website

https://github.com/mylar3/mylar3

### Description

Mylar is an automated Comic Book (cbr/cbz) downloader program for use with NZB and torrents.

Mylar allows you to create a watchlist of series that it monitors for various things (new issues, updated information, etc). It will grab, sort, and rename downloaded issues. It will also allow you to monitor weekly pull-lists for items belonging to said watchlisted series to download, as well as being able to monitor and maintain story-arcs.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by spocoeric** [comment]
With the release of Komga for comic books, it would be nice to have the equivalent *arr, Mylar3.

**Comment by davalanche** [comment]
I can look at this one @MickLesk I am familiar with the application.  Thanks!

**Reply by MickLesk** [reply]
Yes, You're welcome to do so 👍

When you're ready, create a PR.

**Reply by spocoeric** [reply]
Thank you!
